src/main/kotlin/omok/controller/Controller.kt

```diff
@@ -0,0 +1,62 @@
+package omok.controller
+
+import omok.domain.Board
+import omok.domain.HorizontalAxis
+import omok.domain.Player
+import omok.domain.Position
+import omok.domain.Stone
+import omok.domain.judgement.LineJudgement
+import omok.domain.state.State
+import omok.domain.state.Turn
+import omok.domain.state.Win
+import omok.view.GameView
+
+class Controller(private val gameView: GameView) {
+    private val board = Board(Player(), Player())
+    private var lastPosition: String? = null
+
+    fun gameStart() {
+        printStart()
+        turn(Turn.Black)
+    }
+
+    private fun printStart() = gameView.printStartMessage()
+
+    private fun turn(state: State) {
+        return when (state) {
```

Comment:
```suggestion
       when (state) {
```
리턴타입이 없다면, return 문을 작성하지 않으셔도 됩니다. 😄

Reply:
넵! 코드 작성 시 착각을 했던 것 같습니다. 알려주셔서 감사합니다!

Reply:
재귀를 막기 위해서 return 을 사용한 것인데 지워도 코드 진행에 아무 상관 없을까요?!

Reply:
재귀를 막기 위한 처리도 좋지만, 코드의 가독성이 더 중요합니다.
재귀를 막기 위해, 모든 Unit 타입의 함수에 return 문을 사용하지 않으니까요. 😅